wrapping/itkLabelSetMorphBaseImageFilter.wrap

```diff
@@ -1,3 +1,8 @@
 itk_wrap_class("itk::LabelSetMorphBaseImageFilter" POINTER)
-  itk_wrap_image_filter("${WRAP_ITK_SCALAR}" 2 2+)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    foreach(t ${WRAP_ITK_SCALAR})
+      itk_wrap_template("${ITKM_I${t}${d}}TRUE" "${ITKT_I${t}${d}},true,${ITKT_I${t}${d}}")
```

Comment:
To be consistent with the name mangling, we should add the second image type, i.e.

```
${ITKM_I${t}${d}}TRUE
```

to

```
${ITKM_I${t}${d}}TRUE${ITKM_I${t}${d}}
```

and in the line below.

Reply:
Thanks both @phcerdan and @thewtex !